core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -398,6 +399,7 @@ object Boot extends IOApp with LazyLogging {
       val multiCloudWorkspaceServiceConstructor: (UserInfo) => MultiCloudWorkspaceService = MultiCloudWorkspaceService.constructor(
         slickDataSource,
         workspaceManagerDAO,
+        billingProfileManagerClientProvider,
```

Comment:
I wasn't sure if we want to pass `billingProfileManagerDAO` instead of just the client. I need the client to be able to look up the profile during workspace creation.